docs/admin/src/netlify-cms.js

```diff
@@ -1,12 +1,13 @@
 import CMS from 'netlify-cms';
 import { default as componentsPreviewTemplate } from './widgets/componentsPreviewTemplate';
+import { default as genericPreviewTemplate } from './widgets/genericPreviewTemplate';
 import { Preview as navPreview } from './widgets/navigationPreviewTemplate';
 
 CMS.registerPreviewTemplate( 'getting-started', componentsPreviewTemplate );
 CMS.registerPreviewTemplate( 'foundation', componentsPreviewTemplate );
 CMS.registerPreviewTemplate( 'components', componentsPreviewTemplate );
 CMS.registerPreviewTemplate( 'templates', componentsPreviewTemplate );
 CMS.registerPreviewTemplate( 'navigation', navPreview );
-
+CMS.registerPreviewTemplate( 'home', genericPreviewTemplate );
 
 CMS.registerPreviewStyle( '/design-system/dist/css/main.css' );
```

Comment:
Not a change in this PR, but seems like we should set up a variable for the JavaScript paths equivalent to the `{{ site.baseurl }}` variable for asset paths used in the templates.